BeerOverflowWindowsApp/BarProviders/GetBarListFacebook.cs

```diff
@@ -21,52 +21,67 @@ class GetBarListFacebook : IBeerable
         public List<BarData> GetBarsAround(string latitude, string longitude, string radius)
         {
             RegexTools.LocationDataTextIsCorrect(latitude, longitude, radius);
-            var result = GetBarData(latitude, longitude, radius);
-            var barList = FacebookDataToBars(result);
+            var placeList = GetBarData(latitude, longitude, radius);
+            var barList = PlaceListToBarList(placeList);
             return barList;
         }
 
-        private PlacesResponse GetBarData(string latitude, string longitude, string radius)
+        private IEnumerable<Place> GetBarData(string latitude, string longitude, string radius)
```

Comment:
So as I understand, now we have a list of places we've got from providers, yes?

Reply:
Yes. Earlier we had to search in PlacesResponse.Places which was of type List<Place>. I simplified it so it's easier to read and operate.